packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-fulfilment/route.js

```diff
@@ -16,7 +16,10 @@ export const getData = async request => {
     throw new GetDataRedirect(CONTACT.uri)
   }
 
-  return { youOrOther: youOrOther(permission) }
+  return {
+    SHOW_NOTIFICATION_BANNER: process.env.SHOW_NOTIFICATION_BANNER?.toLowerCase() === 'true',
```

Comment:
convention is to use camelCase rather than all caps with underscores - please would you change it?